.circleci/config.yml

```diff
@@ -111,3 +126,7 @@ workflows:
       - lint:
           requires:
             - test
+      - deposit_contract:
+          requires:
+            - install_env
+            - lint
```

Comment:
Why require `lint`?

Reply:
To avoid the race condition of `venv` :'(

I will think more about how to make it in parallel...